index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Linkbrary</title>
+    <meta property="og:title" content="Linkbrary" key="title" />
+    <meta
+      name="description"
+      property="og:description"
+      content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요"
+      key="description"
+    />
+    <meta
+      property="og:image"
+      content="https://visitbusan.net/uploadImgs/files/cntnts/20211130150754165_wufrotr"
+      key="image"
+    />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@linkbrary" />
+    <meta name="twitter:title" content="Linkbrary" />
+    <meta name="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta
+      name="twitter:image"
+      content="https://visitbusan.net/uploadImgs/files/cntnts/20211130150754165_wufrotr"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="./src/landing.css" />
+  </head>
+  <body>
+    <nav>
+      <div class="gnb">
+        <a href="index.html">
+          <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+        </a>
+        <a class="cta cta-short" href="signin.html">
+          <span>로그인</span>
+        </a>
+      </div>
+    </nav>
```

Comment:
nav 태그에 바로 gnb 클래스 적용하면 불필요하게 div로 감싸지 않겠네요~

```suggestion
    <nav class="gnb">
      <a href="index.html">
        <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
      </a>
      <a class="cta cta-short" href="signin.html">
        <span>로그인</span>
      </a>
    </nav>
```